javascript/linters/.eslintrc

```diff
@@ -105,14 +104,14 @@
      */
     "block-scoped-var": 2,
     "consistent-return": 2,
-    "curly": 2,
-    "default-case": 2,
+    "curly": [2, "multi"],
```

Comment:
I prefer to put curly braces all the time because it forces you to put the statement on its own line rather than putting an if and a statement on the same line. This makes it easier to read because you can say "this line is a test", "this line does something".
So I'd rather switch this to:
`"curly": 1`